web/src/components/__tests__/Feedback.spec.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react'
+import { fireEvent, render } from '@testing-library/react'
+import Feedback from '../Feedback'
+import { ThemeProvider } from 'styled-components'
+import { SendingState } from '../FeedbackContainer'
+import buildConfig from '../../constants/buildConfig'
+
+jest.mock('react-i18next')
+
+describe('Feedback', () => {
+  beforeEach(() => {
+    jest.clearAllMocks()
+  })
+
+  const t = (key: string): string => key
+
+  const onCommentChanged = jest.fn()
+  const onContactMailChanged = jest.fn()
+  const onSubmit = jest.fn()
+  const closeFeedbackModal = jest.fn()
+
+  const buildProps = (
+    isPositiveFeedback: boolean,
+    comment: string,
+    isSearchFeedback = false,
+    sendingStatus = SendingState.IDLE
+  ) => {
+    return {
+      comment,
+      isPositiveFeedback,
+      isSearchFeedback,
+      contactMail: 'test@example.com',
+      sendingStatus,
+      onCommentChanged,
+      onContactMailChanged,
+      onSubmit,
+      t,
+      closeFeedbackModal
+    }
+  }
+
+  it('button should be disabled for negative Feedback and no input', () => {
+    const { getByText } = render(
+      <ThemeProvider theme={buildConfig().lightTheme}>
+        <Feedback {...buildProps(false, '')} />
+      </ThemeProvider>
+    )
+    expect(getByText('feedback:send')).toBeDisabled()
+  })
+
+  it('button should be enabled for positive Feedback and no input', () => {
+    const { getByText } = render(
+      <ThemeProvider theme={buildConfig().lightTheme}>
+        <Feedback {...buildProps(true, '')} />
+      </ThemeProvider>
+    )
+    expect(getByText('feedback:send')).not.toBeDisabled()
```

Comment:
is there maybe a isEnabled? :D

tests look good!

Reply:
yes, there is! Changed it :)